src/BizHawk.Emulation.Cores/CPUs/FairchildF8/F3850.Registers.cs

```diff
@@ -174,7 +174,7 @@ public bool FlagICB
 		public bool IRQRequest
 		{
 			get => Regs[IRQR] > 0;
-			set => Regs[IRQR] = value == true ? (byte)1 : (byte)0;
+			set => Regs[IRQR] = value ? (byte)1 : (byte)0;
```

Comment:
This probably doesn't matter here because IRQRequest appears to be unused, but I would expect this to either be on one extreme for performance marking the public bool as unsafe and doing a bit hack `set => Regs[IRQR] = *(byte*)&value;` or for readability and safety `set => Regs[IRQR] = Convert.ToByte(value);`

Reply:
Yeah I'm gonna say no to `unsafe` optimizations on the microscopic level for an unused property here.

Reply:
yeah it looks much preferable to do Convert.ToByte() I just figured it worth mentioning when the code is already using (albeit safe) byte casts

Reply:
These casts are const, and the ternary will probably be optimised by the JITter. (I'd go with the BCL helper still.)